### PR TITLE
Add ConfigChange & ImageChange triggers

### LINKED ISSUE #30: Add ImageChange triggers

### PR DESCRIPTION
**Pull request by jpopelka**
Fixes #30

https://blog.openshift.com/image-streams-faq

- [x] test @ stg
- [x] create a job which will periodically call `oc import-image`, becase OS Online gives a fak about [periodical imports](https://docs.openshift.com/container-platform/3.11/architecture/core_concepts/builds_and_image_streams.html#image-stream-mappings-working-periodic)

### CODE REVIEW

File: openshift/imagestream-worker.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+metadata:
+  name: packit-worker
+spec:
+  tags:
+    - name: {{ deployment }}
+      from:
+        kind: DockerImage
+        name: {{ image_worker }}:{{ deployment }}
+      importPolicy:
+        # periodically query registry to synchronize tag and image metadata
```
Comment (TomasTomecek) [inline openshift/imagestream-worker.yml.j2]: so it will kill pods if it finds newer image?

Comment (jpopelka) [reply openshift/imagestream-worker.yml.j2]: Here, in `ImageStream`, the `importPolicy: scheduled: true` says that if a newer image appears in a registry it is automatically imported into the image stream.

Then in `DeploymentConfig: triggers: imageChangeParams: automatic` says whether to re-deploy (i.e. "kill pods") if a newer image appears in the image stream.

Comment (TomasTomecek) [reply openshift/imagestream-worker.yml.j2]: got it, danke!

Comment (jpopelka) [reply openshift/imagestream-worker.yml.j2]: added to README

**Comment by jpopelka** [comment]
Deployed to `stg` && ready for review. I just need to add a job for periodically reimport new images into image streams, because for some strange reason this does not work on OS Online.

File: README.md [hunk 2]
```diff
@@ -18,81 +18,114 @@
...
+Staging images are `:stg` tagged and built from `master` of `packit` and `packit-service`.
+Production images are `:prod` tagged and built from `stable` branch of `packit` and `packit-service`.
+To move `stable` branch to a newer 'stable' commit:
+- git branch -f stable commit-hash
+- git push [-u upstream] stable
 
+Beware: [packit-service-worker image](https://cloud.docker.com/u/usercont/repository/docker/usercont/packit-service-worker) is not automatically rebuilt when its base [packit image](https://cloud.docker.com/u/usercont/repository/docker/usercont/packit) changes. You have to [trigger](https://cloud.docker.com/u/usercont/repository/docker/usercont/packit-service-worker/builds) the build manually.
 
-### Updating
+### deploymentConfig vs. StatefulSet
```
Comment (TomasTomecek) [inline README.md]:
```suggestion
### DeploymentConfig vs. StatefulSet
```

**Pull request merged**